docs/src/user/manual/explore/charts.md

```diff
@@ -15,40 +15,192 @@ context_id: ZSVG5K6yFRsQVQF5WQFqJR
 
 The `Charts` tab is a powerful tool for visualising data through various charts and graphs. This documentation outlines the functionality of the `Download Data Visualisation` feature, which allows the user to download the charts presented on the page.
 
-## Charts Page
+## Charts Tab
 
 ![Charts Page](./img/charts-1.png)
 
-1. **Explore**: By clicking on the `EXPLORE` on the navigation bar the user will be able to see the option for the Charts.
+1. **Explore**: To access the Charts, click on the `EXPLORE` option in the navigation bar.
 
-2. **Charts**: By clicking on the `CHARTS` tab, the user will be able to see the various charts. The `Charts` tab offers a variety of charts for data visualisation.
+2. **Charts**: Navigate to the `CHARTS` tab to explore a variety of charts available for data visualisation.
+
+3. **Filters**: Enhance chart visualisation by utilising the available filters for more precise data representation.
+
+4. **Clear All**: Click on `Clear All` to remove all applied filters and reset the chart view.
+
+5. **Message**: A helpful and informative message prompts users to explore and view charts.
 
 3. **Download Data Visualisation**: The `DOWNLOAD DATA VISUALISATION` button is a convenient feature that enables the user to download the charts and visualisations displayed on the page.
```

Comment:
This needs to be updated otherwise the new `sane_lists` mkdocs plugin will render it as `3`
```suggestion
6. **Download Data Visualisation**: The `DOWNLOAD DATA VISUALISATION` button is a convenient feature that enables the user to download the charts and visualisations displayed on the page.
```